error: implicit conversion from 'unsigned long' to 'float' changes value from 281474976710655 to 281474976710656

OS: FreeBSD 12 amd64
Rev. ```1.16.3-784-g981ecfca```

```
/wrkdirs/usr/ports/net-im/qTox/work/qTox-1.16.3-784-g981ecfca/src/widget/tool/identicon.cpp:112:75: error: implicit conversion from 'unsigned long' to 'float' changes value from 281474976710655 to 281474976710656 [-Werror,-Wimplicit-int-float-conversion]
           / (((static_cast<uint64_t>(1)) << (8 * IDENTICON_COLOR_BYTES)) - 1);
           ~  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~
1 error generated.
```

Log: http://beefy18.nyi.freebsd.org/data/head-amd64-default/p528397_s358971/logs/qTox-1.16.3.784.log (IPv6 URL)

--- a/src/widget/tool/identicon.cpp
+++ b/src/widget/tool/identicon.cpp
@@ -109,7 +109,7 @@ float Identicon::bytesToColor(QByteArray bytes)
 
     // normalize to 0.0 ... 1.0
     return (static_cast<float>(hue))
-           / (((static_cast<uint64_t>(1)) << (8 * IDENTICON_COLOR_BYTES)) - 1);
+           / (static_cast<float>(((static_cast<uint64_t>(1)) << (8 * IDENTICON_COLOR_BYTES)) - 1));
 }
 
 /**

